package.json

```diff
@@ -1,32 +1,16 @@
 {
-	"name": "astro-flow-root",
-	"type": "module",
-	"version": "0.3.6",
-	"workspaces": [
-		"demo",
-		"packages/*"
-	],
```

Comment:
This changes the project from being a monorepo to _not_ a monorepo, which makes it a lot harder to test the demo.

Reply:
The code that is in there is moved to `pnpm-workspace.yaml`
 ```json
"workspaces": [
		"demo",
		"packages/*"
	],
```

Look in `pnpm-workspace.yaml` for this.
```yaml
packages:
  - "packages/**"
  - "demo/**"

```